### PR TITLE
Basic support for Asterisk 13

### PR DESCRIPTION
**Pull request by polysics**
Starting work to make Punchblock compatible with Asterisk 13. The first step is handling the new AGIAsync events correctly.

### CODE REVIEW

File: spec/punchblock/translator/asterisk_spec.rb [hunk 1]
```diff
@@ -415,6 +419,29 @@ module Translator
             subject.handle_ami_event ami_event
           end
 
+          context 'with an Asterisk 13 AsyncAGIStart event' do
+            let :ami_event do
+            RubyAMI::Event.new 'AsyncAGIStart',
```
Comment (benlangfeld) [inline spec/punchblock/translator/asterisk_spec.rb]: Indentation.

**Comment by polysics** [comment]
This should be what is needed to get Asterisk 13 AsyncAGI events supported.

**Comment by benlangfeld** [comment]
@polysics Please base further changes on this branch and do pull requests to this branch instead of to develop. We'll use this branch/PR to aggregate the full set of changes for basic Asterisk 13 compatibility.

**Comment by polysics** [comment]
Aside from the two failing tests, which I do not have locally, the main issue right now seems to figure out the different response format and chain of events for Asterisk 11 vs. 13 to get a call into a controller.
Attached is what I see currently. The `@components` list is empty when the AsyncAGIExec command comes in, so nothing happens.

**Comment by benlangfeld** [comment]
Those test failures appear to be specific to CI.

**Pull request merged**